newapi/tests/conftest.py

```diff
@@ -101,20 +108,22 @@ def setup_database_part_1():
 
 @pytest.fixture(scope="session")
 def checkout_pipeline(tmpdir_factory):
-    if not pytest.create_db:
+    """Clone pipeline repo to then run fastpath from S3 and citizenlab importer"""
+    if not pytest.create_db and not pytest.inject_msmts:
         return
     d = tmpdir_factory.mktemp("pipeline")
     if d.isdir():
         shutil.rmtree(d)
     # cmd = f"git clone --depth 1 https://github.com/ooni/pipeline -q {d}"
-    cmd = f"git clone --depth 1 https://github.com/ooni/pipeline --branch stop-after -q {d}"
+    # FIXME change branch
+    cmd = f"git clone --depth 1 https://github.com/ooni/pipeline --branch clickhouse2 -q {d}"
```

Comment:
Since the clickhouse2 branch has been merged into master and deleted, this needs to be changed in order for the CI tests to run properly.

Reply:
Fixed, good spot!